spec/requests/api/v1/registrations_controller_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::RegistrationsController, type: :request do
+  describe 'POST#signup' do
+    context 'given valid email and password' do
+      application ||= Fabricate(:application)
+      valid_signup_params = {
+        email: "test+#{Time.now.to_i}@nimblehq.co",
+        password: '123456',
+        password_confirmation: '123456',
+        client_id: application.uid
+      }
+      it 'returns http status of created' do
+        post api_v1_registrations_path, params: valid_signup_params
+
+        expect(response).to have_http_status(:created)
+      end
```

Comment:
Please adjust the other tests below as well. For preparing test data, it should be included within a specific `it ...`

```suggestion
      it 'returns the created status' do
        application ||= Fabricate(:application)
        valid_signup_params = {
          email: "test+#{Time.now.to_i}@nimblehq.co",
          password: '123456',
          password_confirmation: '123456',
          client_id: application.uid
        }

        post api_v1_registrations_path, params: valid_signup_params

        expect(response).to have_http_status(:created)
      end
```